README.md

```diff
@@ -243,91 +256,397 @@ The SDK provides a simple interface using the `.upload()` method to upload files
 The `upload()` method requires at least the `file` and the `fileName` parameter to upload a file. You can pass other parameters supported by the ImageKit upload API using the same parameter name as specified in the upload API documentation. For example, to specify tags for a file at the time of upload, use the `tags` parameter as specified in the [documentation here](https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload).
 
 Sample usage
+
 ```cs
-ImagekitResponse resp = await imagekit
-    .FileName("my_file_name.jpg")
-    .UploadAsync(<fullPath|url|base_64|binary>);
-```
+ // Upload By URI
+FileCreateRequest request = new FileCreateRequest
+{
+file = "http://www.google.com/images/logos/ps_logo2.png",
```

Comment:
Huge inconsistency between parameter names between readme and sample program/test cases.
`FileName` is no where in sample and test cases now. The whole readme is outdated it seems. So I am not adding any more comments in readme.